css-three-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+h3,
+h2 {
+  font-family: Lato, sans-serif;
+  color: white;
+}
+
+h3 {
+  font-size: 20px;
+  line-height: 0em;
+  padding-top: 16px;
+}
+
+p {
+  font-family: Lato, sans-serif;
+  font-size: 14px;
+  line-height: 1.5em;
+  color: #d8d9da;
+  padding: 0 29px;
+}
+
+body {
+  background-color: #219bdd;
+}
+
+.container {
+  width: 900px;
+  margin: 50px auto 0;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
The `row` class should only be `display: flex`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

Reply:
Fixed the .row and .column classes 
![css-three-column-layout-02](https://user-images.githubusercontent.com/96809909/151646607-aedd5dfe-b292-42b7-ba60-70c258dd356d.png)